pg_query.h

```diff
@@ -19,8 +19,18 @@ typedef struct {
   PgQueryError* error;
 } PgQueryNormalizeResult;
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 void pg_query_init(void);
 PgQueryNormalizeResult pg_query_normalize(char* input);
 PgQueryParseResult pg_query_parse(char* input);
+void pg_query_normalize_free(PgQueryNormalizeResult result);
+void pg_query_parse_free(PgQueryParseResult result);
```

Comment:
Should we maybe call them `pg_query_free_normalize_result` and `pg_query_free_parse_result` instead?

(but also fine with the suffix variant, no strong opinion here)